docs/lab/overview/insurance.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Insurance Ordering"
+---
+
+Vital supports ordering with commercial insurance billing. To set this up, there are a set of steps required. This document explains the complete flow for ordering via commercial insurance.
+
+
+## Setting Insurance Data
+
+To place an order for a `user` in Vital with commercial insurance, you must first provide Vital with insurance data.
+
+This can be done via the [`POST /{user_id}/insurance`](/api-reference/user/create-insurance) endpoint, with the following payload:
+
+<Note>
+Note that you can only set insurance data for users that already have patient information set, either via an order or via the [`PATCH /v2/user/{user_id}/info`](/api-reference/user/upsert-info).
+</Note>
+
+```json
+"insurance": {
+    "payor_code": "string",
+    "member_id": "string",
+    "group_id": "string", // optional
+    "relationship": "Self, Spouse, Other",
+    "insured": {
+        "first_name": "string",
+        "last_name": "string",
+        "dob": "YYYY-MM-DD",
+        "gender": "string",
+        "address": {
+            "first_line": "string", 
+            "second_line": "string", // optional
+            "zip_code": "string",
+            "state": "string",
+            "city": "string",
+        },
+        "phone_number": "string",
+        "email": "string"
+    },
+    "guarantor": {
+        "first_name": "string",
+        "last_name": "string",
+        "dob": "YYYY-MM-DD",
+        "gender": "string",
+        "address": {
+            "first_line": "string", 
+            "second_line": "string", // optional
+            "zip_code": "string",
+            "state": "string",
+            "city": "string",
+        },
+        "phone_number": "string",
+        "email": "string",
+    } // optional if relationship is Self
```

Comment:
Is it optional or what would happen if I set a different guarantor than myself but set relationship to self? Will it be ignored? If so I would make that explicit.